pkg/kubectl/history.go

```diff
@@ -102,12 +102,12 @@ type DeploymentHistoryViewer struct {
 // ViewHistory returns a revision-to-replicaset map as the revision history of a deployment
 // TODO: this should be a describer
 func (h *DeploymentHistoryViewer) ViewHistory(namespace, name string, revision int64) (string, error) {
-	versionedExtensionsClient := h.c.ExtensionsV1beta1()
-	deployment, err := versionedExtensionsClient.Deployments(namespace).Get(name, metav1.GetOptions{})
+	versionedAppsClient := h.c.AppsV1()
```

Comment:
Have you checked if these commands work fine with previous version of the server, it should but I'm asking to have that double checked.

Reply:
I checked a build of kubectl from this PR against a v1.9.3 cluster, and the following commands worked:

```
kubectl edit deployment <x> (to create history)
kubectl rollout history deployment/<x>
kubectl rollout undo deployment/<x>
kubectl rollout status deployment/<x>
```

We also have automated kubectl skew tests that should run post-submit.

As expected, I observed that the rollout commands *do not* work against a v1.8.9 cluster, because apps/v1 did not exist until v1.9.0. This is fine because this change will roll out with kubectl v1.11 at the earliest, and v1.8 clusters are outside the client/server compatibility window for that release.